### PR TITLE
feat: [CX-2568]: Adding visual clue to the insights tab after adding an artwork with insights

### PR DESCRIPTION
**Pull request by MrSltun**
This PR resolves [CX-2568]

### Description
This PR adds the logic for showing visual clues on the StickyTabPage tabs and the logic for displaying a visual clue on the insights tab inside My Collection when adding artwork with insights to my collection

### Pictures and Videos
**_New Video_**

https://user-images.githubusercontent.com/20655703/171399598-969d64a2-f0d3-4dfd-8f4e-18a5faf9c185.mp4

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [x] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [x] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- adding a visual clue to the insights tab when adding artwork with insights - mrsltun

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[CX-2568]: https://artsyproduct.atlassian.net/browse/CX-2568?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/store/config/visualClues.ts [hunk 1]
```diff
@@ -25,6 +25,9 @@ export const visualClues = defineVisualClues({
     description: "The message shown if the collector profile is incomplete",
   },
   AddArtworkWithoutInsightsMessage_MyCTab: {
-    description: "The message shown after addind an artwork without insights for the Insights tab",
+    description: "The message shown after adding an artwork without insights for the Insights tab",
+  },
+  AddArtworkWithInsights: {
+    description: "The visual cue shown on the Insights tab after adding an artwork with insights",
```
Comment (MounirDhahri) [inline src/app/store/config/visualClues.ts:31]:
```suggestion
    description: "The message shown on My collection Insights tab after adding an artwork with insights",
```

File: src/app/store/config/visualClues.ts [hunk 2]
```diff
@@ -25,6 +25,9 @@ export const visualClues = defineVisualClues({
     description: "The message shown if the collector profile is incomplete",
   },
   AddArtworkWithoutInsightsMessage_MyCTab: {
-    description: "The message shown after addind an artwork without insights for the Insights tab",
+    description: "The message shown after adding an artwork without insights for the Insights tab",
```
Comment (MounirDhahri) [inline src/app/store/config/visualClues.ts:28]:
```suggestion
    description: "The message shown on My Collection Insights tab after adding an artwork without insights",
```

**Review by MounirDhahri — APPROVED** [review]

**Comment by MounirDhahri** [comment]
I will merge this and create a new build, let me know if you notice something wrong @olerichter00 
Also, I think we need to add tests to form

**Comment by olerichter00** [comment]
🙌

File: src/app/Scenes/MyCollection/Screens/ArtworkForm/MyCollectionArtworkForm.tsx [hunk 3]
```diff
@@ -382,11 +382,12 @@ const addArtworkMessages = async ({
...
-      sourceTab === Tab.collection
-        ? "AddedArtworkWithInsightsMessage_MyCTab"
-        : "AddedArtworkWithInsightsMessage_InsightsTab"
-    )
+    if (sourceTab === Tab.collection) {
+      addClue("AddedArtworkWithInsightsMessage_MyCTab")
+      addClue("AddArtworkWithInsights")
+    } else {
+      addClue("AddedArtworkWithInsightsMessage_InsightsTab")
+    }
```
Comment (olerichter00) [inline src/app/Scenes/MyCollection/Screens/ArtworkForm/MyCollectionArtworkForm.tsx:390]: 👀

Comment (MrSltun) [reply src/app/Scenes/MyCollection/Screens/ArtworkForm/MyCollectionArtworkForm.tsx:390]: this one is to add the visual clue on the Insights tab when there are insights for the newly added artwork, had to refactor it to add 2 clues 🌚

**Pull request merged**